Index: php-5.4.11/TSRM/configure.in
===================================================================
--- php-5.4.11.orig/TSRM/configure.in	2013-02-15 22:26:01.000000000 +0000
+++ php-5.4.11/TSRM/configure.in	2013-02-15 23:07:39.000000000 +0000
@@ -17,9 +17,6 @@
 TSRM_THREADS_CHECKS
 	
 AM_PROG_LIBTOOL
-if test "$enable_debug" != "yes"; then
-  AM_SET_LIBTOOL_VARIABLE([--silent])
-fi
 
 dnl TSRM_PTHREAD
 
Index: php-5.4.11/configure.in
===================================================================
--- php-5.4.11.orig/configure.in	2013-02-15 23:06:51.000000000 +0000
+++ php-5.4.11/configure.in	2013-02-15 23:07:39.000000000 +0000
@@ -1399,9 +1399,6 @@
 ])
 AC_PROG_LIBTOOL
 
-if test "$enable_debug" != "yes"; then
-  PHP_SET_LIBTOOL_VARIABLE([--silent])
-fi
 
 dnl libtool 1.4.3 needs this.
 PHP_SET_LIBTOOL_VARIABLE([--preserve-dup-deps])
